PriceCalculatorKata/Tax.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace PriceCalculatorKata.Tax
+{
+    public interface ITax
+    {
+        float getTax();
```

Comment:
`    public interface ITax
    {
        float Amount { get; }
    }`
isn't this simpler?